gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/JDBCKnoxShellTableBuilder.java

```diff
@@ -42,16 +40,16 @@ public JDBCKnoxShellTableBuilder username(String username) {
     return this;
   }
 
-  public String username() {
-    return username;
-  }
-
   public JDBCKnoxShellTableBuilder pwd(String pass) {
     this.pass = pass;
     return this;
   }
 
-  public String password() {
+  public String username() {
+    return username;
+  }
+
+  public String pwd() {
```

Comment:
`password()` was changed to `pwd`? Any reason?

Reply:
A result of recovering from IDE local history after a problematic merge.
Changing back to password().

Reply:
Looks like this is still `pwd()` instead of `password()` in the recent changes?